package.json

```diff
@@ -174,6 +175,7 @@
     "qr-code-component-ng": "0.0.4",
     "rxjs": "5.5.12",
     "text-encoding": "0.6.4",
+    "text-mask-core": "^5.1.2",
```

Comment:
This library is no longer maintained since 2 years. I'd fix version to "5.1.2", without `^`